spec/latest/core/index.html

```diff
@@ -1193,6 +1193,36 @@ <h3>Templated Links</h3>
       </dl>
     </aside>
 
+    <p>The example <i>Description of an IRI Template</i> that was already mentioned
+      uses a <i>variableRepresentation</i> on the whole <i>IriTemplate</i>, but
+      each mapped variable represented as an <i>IriTemplateMapping</i> can have
+      it's own <i>variableRepresentation</i> declaration which SHOULD override
```

Comment:
```suggestion
      its own <i>variableRepresentation</i> declaration which MUST override
```

Or simply `which overrides`?